<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PairedSetCreator</class>
 <widget class="QWidget" name="PairedSetCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>501</width>
    <height>442</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create a paired set</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="setNameBox">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name of the set:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="setName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="t1Lbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Term 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="termOneEntry">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="t2Lbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Term 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="termTwoEntry">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="flagged">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Flag for revision?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SavePairChanges">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save changes to term</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DeleteTerm">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Delete term</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="Terms">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Terms in the set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveSet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateNewTerm">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create new term</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
